packages/xo-web/src/common/intl/messages.js

```diff
@@ -2408,15 +2408,15 @@ const messages = {
 
   // Licenses
   allHostsMustBeBound: 'All hosts must be bound to a license',
-  bound: 'Bound',
+  boundSelectLicense: 'Bound (Plan (ID), expiration date, host - pool)',
```

Comment:
It's not exactly a "plan" in case of proxies but let's leave it like that for now.